src/test/dragons.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/205458954-ff9cc12c-6953-496e-b4e4-a15272ead5b4.PNG)

**[Optional]**
- [ ] Your tests are running and passing but they are showing these warnings in the console. It would be nice if you resolve them too 👍